src/Engine/Sound/SoundManager.cpp

```diff
@@ -258,6 +291,28 @@ bool SoundManager::OnPlayBackgroundMusic(const Events::PlayBackgroundMusic & e)
     return true;
 }
 
+
+bool SoundManager::OnPlayAnnouncerVoice(const Events::PlayAnonuncerVoice& e)
+{
+    auto listenerComponents = m_World->GetComponents("Listener");
+    for (auto it = listenerComponents->begin(); it != listenerComponents->end(); it++) {
+        if ((*it).EntityID != m_LocalPlayer.ID) {
+            break;
```

Comment:
This will break if the first listnerComponent isen't attached to the m_LocalPlayer and then continue to loop if m_LocalPlayer was found. Am i wrong in suggesting a continue if it isen't m_LocalPlayer and a break at the end if we find m_LocalPlayer?